static/js/script.js

```diff
@@ -11,7 +11,13 @@ function createLayout() {
     const gridRows = 9
     const gridCols = 9
 
-    let cachedMatrix = JSON.parse(localStorage.getItem("matrix")) // load puzzle from cache or generate?
+    let cachedMatrix;
+    try {
+        cachedMatrix = JSON.parse(localStorage.getItem("matrix")); 
+    } catch (e) {
+        console.log('Error trying to load saved matrix from localStorage', e.message);
```

Comment:
Could send user an Alert message, or...POST msg back to server so Dev can see what issues users have